indexes/zk_ee/ethereum-general-state-tests.yaml

```diff
@@ -413,9 +413,8 @@ entries:
           creationTxInitCodeSizeLimit.json:
             hash: '0xc70a4a5673cb5330590a49e903a2666d'
             enabled: true
-            # Skipped for Emulator
-            # skipCases: ["valid"]
-            # comment: "Can't simulate refund for creation now"
+            skipCases: ["valid"]
+            comment: "Can't simulate refund for creation now"
```

Comment:
Can you explain a little?

Reply:
Ah, bruh, I forgot this case from Emulator. What is it about?

Reply:
It's just that we don't have refunds for now in ZK OS